tests/zfs-tests/tests/functional/rsend/send_encrypted_props.ksh

```diff
@@ -179,11 +181,25 @@ recv_cksum=$(md5digest /$ds/$TESTFILE0)
 log_must test "$recv_cksum" == "$cksum"
 log_must zfs destroy -r $ds
 
+# Test that we can override an unencrypted, incremental, properties stream's
+# encryption settings, receiving it as an encrypted child.
+log_note "Must be able to receive incremental stream with props to encrypted"
+ds=$TESTPOOL/crypt/recv
+log_must eval "zfs send -p $snap > $sendfile"
```

Comment:
The code looks good, but the test case may need to be updated: `zfs send -p $snap` is not incremental.

Reply:
Its actually just the comment that's wrong (and git made it confusing). The new test case is the second one, which is largely the same as the first, except that it uses the second snapshot. I will fix the comment now.

Reply:
Hum, now i see we are using `zfs send -R` instead of `zfs send -p` to generate an incremental send stream, which works but is kind of confusing: i would suggest using `zfs send -i` explicitly to generate an incremental stream.

Reply:
I wanted to do it like this because (1) it is a more complicated case to handle and (2) the problem was originally reported as a failure for `zfs send -R` to work incrementally and basically worked as it did in the test case.